gobblin-runtime/src/main/java/org/apache/gobblin/service/monitoring/FlowStatusGenerator.java

```diff
@@ -45,14 +46,14 @@
    * @param count
    * @return the latest <code>count</code>{@link FlowStatus}es. null is returned if there is no flow execution found.
    */
-  public List<FlowStatus> getLatestFlowStatus(String flowName, String flowGroup, int count) {
+  public List<FlowStatus> getLatestFlowStatus(String flowName, String flowGroup, int count, String tag) {
```

Comment:
Can we update the javadoc to include the tag param? Also, maybe good to explain how the tag field can be used.